### ISSUE TITLE
Can't drag more objects into a Part:Compound

### ISSUE THREAD

**Issue by theosib** [FreeCAD/FreeCAD#16167]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

If I have a Union in Part, I can drag more objects into it.

But this doesn't work for Compound. Dragging something into the compound has no effect.

This seems like an inconsistency.

### Full version info

```shell
OS: macOS 14.6.1
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.38553 (Git)
Build type: Release
Branch: main
Hash: 59c1ccec3e6b70f56eeee8f94d361019b84bd850
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: English/United States (en_US) [ OS: C/Default (C) ]
Installed mods: 
  * Curves 0.6.44
```

### Subproject(s) affected?

Part

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Roy-043** [comment]
A Part Compound is a low level container. You can only drop objects with a shape. You cannot drop Std_Groups or Std_Parts for that reason.

**Comment by luzpaz** [comment]
Is this a FR then ? or by design and the UI should let user know this is by design ?

**Comment by theosib** [comment]
I was trying to add a simple extruded sketch, but dragging it to the compound failed silently. There wasn't even a console message.

Incidentally, there ought to be some means to remove an element from a container. Deleting it is fine if it's the only instance, but if an object is used multiple places, we don't want to actually delete it.

**Comment by Roy-043** [comment]
That is indeed a special case. You can drop an individual sketch, you can drop the Part Extrude, but not both.

**Comment by Roy-043** [comment]
> Incidentally, there ought to be some means to remove an element from a container.

You can drag objects from containers and drop them on the document node.

**Comment by theosib** [comment]
> > Incidentally, there ought to be some means to remove an element from a container.
> 
> You can drag objects from containers and drop them on the document node.

I actually tried that, and it didn't work. For example, I tried to drag the object below the compound or between the compound and the object above it. None of these things worked.

**Comment by theosib** [comment]
> That is indeed a special case. You can drop an individual sketch, you can drop the Part Extrude, but not both.

I actually just tried to drag the extrude. The sketch is just linked in the extrude.

**Comment by Roy-043** [comment]
> None of these things worked.

The document node is a single entry in the Tree view. It is the entry with this icon: https://wiki.freecad.org/images/d/df/Document.svg

**Comment by theosib** [comment]
Yeah, I was trying to drag it between two other objects at the top level. But it'll only give me one of those "I'll insert here" lines if I'm all the way at the top. I could see this being a problem if there are too many objects in the main document so that the document object is not on screen.

**Comment by FEA-eng** [comment]
So what would be the steps to reproduce it? What kind of object should I try dragging into both containers? So far, I don't see any difference between them in this regard.

```
OS: Windows 10 build 19045
Architecture: x86_64
Version: 1.1.0dev.14555 (Git shallow)
Build date: 2025/10/07 22:49:55
Build type: Release
Branch: main
Hash: 68f083a6d169ec523fa9d46b7cc40addb1e4ffbe
Python 3.11.13, Qt 6.8.3, Coin 4.0.3, Vtk 9.3.1, boost 1_86, Eigen3 3.4.0, PySide 6.8.3
shiboken 6.8.3, xerces-c 3.3.0, IfcOpenShell 0.8.2, OCC 7.8.1
Locale: Polish/Poland (pl_PL)
Navigation Style/Orbit Style/Rotation Mode: CAD/Rounded Arcball/Window center
Stylesheet/Theme/QtStyle: FreeCAD.qss/FreeCAD Light/
Logical DPI/Physical DPI/Pixel Ratio: 96/90.0633/1
```
